bang/templates/include/costumeTopIllustration.html

```diff
@@ -0,0 +1,173 @@
+{% load tools %}
+{% load i18n %}
+
+{% with costume=item %}
+<p id="noWebGLMessage" class="alert alert-danger" style="text-align: left; display:none;">Sorry, a browser that supports WebGL is required to view Live2D models.</p>
+<p id="loadingMessage" class="text-center fontx2 padding20"><i class="flaticon-loading"></i></p>
+<div id="dlcontainer">
+    <canvas id="dltarget" width="800" height="800" style="max-width:100%; max-height:100%;"></canvas>
+
+    <div id="dlcontrols" class="dlhidden" style="display: none;">
+        <button class="btn btn-main btn" title="{% trans "Controls" %}" onclick="DLToggleControls(event)"><i class="flaticon flaticon-settings"></i></button>
+
+        <div class="dldevelopercontrols" style="display: none;">
```

Comment:
You can do:

```html
{% if user.is_staff %}
<div class="dldevelopercontrols" class="staff-only">
    ...
</div>
{% endif %}
```

That's the thing that makes the buttons appear/disappear when you click on the crown button, for staff only.

Reply:
The only "developer control" currently is a file upload for loading custom zips (strictly client-side). This might be useful for regular users (e.g. you can steal animations from other models) so I didn't hide it behind a staff account. It's still hidden by default - the code for enabling it is in the static PR I made